package.json

```diff
@@ -12,8 +12,8 @@
   "files": [
     "dist"
   ],
-  "main": "dist/cjs/index.js",
-  "module": "dist/esm/index.js",
+  "main": "dist/cjs/lib/index.js",
```

Comment:
This solves the problem, but what we should really need to do is to remove lib folder. We don't have any other code apart from the library one, and `src/` folder doesn't have specific files.

Reply:
done!